docs/why-a-workspace.md

```diff
@@ -0,0 +1,69 @@
+# Nx Workspace
```

Comment:
I would write a paragraph about how pretty much every large company works this way (Google, FAcebook, Uber, Twitter). It's important to convey that this approach isn't new. It's being tested.

Reply:
Added

Reply:
I like it